lib/main_containers.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: module loader of container tests
+# Maintainer: qa-c@suse.de
+
+package main_containers;
+use base 'Exporter';
+use Exporter;
+use utils;
+use version_utils;
+use main_common qw(loadtest boot_hdd_image);
+use testapi qw(check_var get_required_var get_var);
+use Utils::Architectures;
+use strict;
+use warnings;
+
+
+our @EXPORT = qw(
+  is_container_test
+  load_container_tests
+);
+
+sub is_container_test {
+    return (check_var('EXTRATEST', 'containers'));
```

Comment:
I was thinking that we don't even need this variable.. we could do something like:
```
get_var('CONTAINER_RUNTIME') ? return 1 : return 0;
```